src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -14,6 +14,8 @@
     <Company>Microsoft</Company>
     <PackageLicenseUrl>https://aka.ms/AzureAppConfigurationDotnetCorePublicPreviewEula</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/Azure/AppConfiguration</PackageProjectUrl>
+    <PackageReleaseNotes>For the latest release notes visit https://aka.ms/MicrosoftFeatureManagementReleaseNotes</PackageReleaseNotes>
+    <PackageTags>Microsoft FeatureManagement FeatureFlags AzureAppConfiguration ASPNETCore</PackageTags>
```

Comment:
>ASPNETCore [](start = 80, length = 10)

I'm not sure if casing matters for existing tags, but I've seen "aspnetcore" in lower case for all packages.
Example: [https://www.nuget.org/packages/Microsoft.AspNetCore.Http](https://www.nuget.org/packages/Microsoft.AspNetCore.Http)

Reply:
Cool, I'll update it.

Reply:
Updated.